gradle.properties

```diff
@@ -1 +1,2 @@
 kotlin.code.style=official
+org.gradle.parallel=true
```

Comment:
I guess this causes the build to fail because multiple projects share the same .nodejs folder and now with parallel they get into conflicts. You can change this in build.gradle => search for workDir, make it per project
Might be you have to do the same for the npmWorkDir